LibTest/js_interop/ExternalDartReference/externalDartReference_t01.dart

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion An opaque reference to a Dart object that can be passed to
```

Comment:
nit: Do we need to copy the documentation here and elsewhere? If it ever changes this might become stale.